app/src/main/resources/openapi_etor.yaml

```diff
@@ -83,6 +95,8 @@ components:
         patientId:
           type: string
           example: MRN7465737865
+    BadRequestError:
+      description: Bad Request
```

Comment:
We return a JSON response body, right?  With an `error` field, yes?  If so, we should flesh this out with those details.  See the above `OrdersResponse` on how to do this.